utilities/README.md

```diff
@@ -0,0 +1,5 @@
+# Assay.Works Instructions
+
+1. Ensure `service-account.json` key is found within the same directory where script is run (becomes available after infrastructure steps are taken with Terraform).
+1. Make changes to `gsutil rsync <source data> gs://waylab-assayworks-bucket/` line to specify the \<source data\> data location to be uploaded to bucket and save the file.
+1. Run the `gsutil_sync.sh` script (for example: `sh ./gsutil_sync.sh`).
```

Comment:
Just to keep things simple, you could create an empty folder in the script's folder where they can put their files and preconfigure the script to upload that folder's contents, rather than have them edit the script. You can put an empty `.gitkeep` file in the empty folder so that it ends up in source control. Also, I'd remove the references to terraform, since they won't be seeing the terraform parts anyway. You could include the bit about changing the upload directory to a different folder in a set of notes that are separate from the basic instructions.

Reply:
Great point, thank you! I've made an update to the script and added some changes to the docs for Assay.Works.